### DISCUSSION TITLE
Sorting and filtering in Search preview

### DISCUSSION THREAD

**Discussion by bsurai** [meilisearch/product#502] [Feedback & Feature Proposal]
At the moment we are allowed to select an index and to enter a search text in the search preview. It would be convenient to be able to set in UI client:
- order, if an index has sortable attributes;
- filters, if an index has filterable attributes.

**Comment by gmourier** [comment]
Hi @bsurai,

Thanks for the suggestion; I've transferred it to the right repository. https://github.com/meilisearch/mini-dashboard/issues/305
